src/dft/backends/rocfft/CMakeLists.txt

```diff
@@ -41,10 +41,22 @@ target_include_directories(${LIB_OBJ}
 target_compile_options(${LIB_OBJ} PRIVATE ${ONEMKL_BUILD_COPT})
 
 find_package(HIP REQUIRED)
-find_package(rocfft REQUIRED)
+find_package(rocfft 1.0.21 REQUIRED)
 
 target_link_libraries(${LIB_OBJ} PRIVATE hip::host roc::rocfft)
 
+# Allow to compile for different ROCm versions.
+# Starting ROCm >=6.0 the include files are one directory level deeper.
```

Comment:
I am worry this might be interpreted as ROCm >= 6.0 is supported which is not currently since we cannot test.

Reply:
I have clarified this in https://github.com/oneapi-src/oneMKL/pull/471/commits/6722aa501f864ec70ab036faebda1bad3f3ee1b7